cerebrium/development/serve.mdx

```diff
@@ -0,0 +1,73 @@
+---
+
+When you are developing a `cortex` deployment on **Cerebrium**, waiting for a build to complete can be time-consuming. To speed up your development process, you can use the `cerebrium serve` command to rapidly iterate on your deployment.
+
+This allows you to run your deployment on a dedicated server, and see the results of your changes in a few seconds.
+
+<Note>
+  This feature is currently in beta and is available to all users. As such, you
+  may encounter bugs or limitations. We are actively working on improving the
+  experience and adding more features. If you have any feedback or suggestions,
+  we'd love to hear from you on discord or slack!
+</Note>
+
+## Usage
+
+To start a served instance, first navigate to the root folder of your **cortex deployment**.
+
+Then, simply run the following command in your terminal:
+
+```bash
+cerebrium serve start
+```
+
+After running the `cerebrium serve start` command, you will see output in your terminal indicating that the server is starting up. This output will include a URL that you can use to access your deployment locally.
+
+Here are some additional commands that can be useful when working with `cerebrium serve`:
+
+To stop a served instance, you can simply press `Ctrl+C` in the terminal where the server is running. This will gracefully shut down the server and terminate the served instance.
+
+When a file changes in your served instance, we'll automatically sync the changes to the server and apply the changes!
+![Example changing code with serve](/images/serve/ModifyingServeCode.gif)
+
+### Testing inference on the served instance
+
+#### Using the local REST server
+
+When you run `cerebrium serve start`, the CLI will print a local REST server address that you can use to send requests to your deployment.
+
+To use the local server to test inference, you can send a **POST** request to: `localhost:8000/predict`
+and the body of the request should be a json object with the input data for your deployment.
+
+Here is an example curl command:
+
+```bash
+curl -X POST http://localhost:8000/predict -H "Content-Type: application/json" --data '{"prompt": "this is my input data"}'
+```
+
+![Sending a prediction request to a serve instance using a curl](/images/serve/ServeCurl.gif)
+
+Since we are using a local api server, you don't need to worry about providing an api key or any other authentication details.
+Otherwise, you can interact with this endpoint as you would with a normal Cerebrium endpoint.
+Your served instance will be running on port 8000 by default, however, you can change this by using the `--port` flag when you start a served instance
+
+Additionally, if you have more than one serve instance running, you can specify the id of the serve instance you want to send the request by using the following pattern: `localhost:8000/{serve_id}/predict` and we'll make sure the request is sent to the correct instance.
+
+## How it works
```

Comment:
I would take out the how it works - didn't really add value if you just explain the above sequentially